packages/cli/src/environments/versionControl/versionControlHelper.ts

```diff
@@ -0,0 +1,53 @@
+import Container from 'typedi';
+import { License } from '../../License';
+import { generateKeyPairSync } from 'crypto';
+import sshpk from 'sshpk';
+import type { KeyPair } from './types/keyPair';
+
+export function isVersionControlLicensed() {
+	const license = Container.get(License);
+	return license.isVersionControlEnabled();
```

Comment:
This got a bit confusing, we're using `licensed` and `enabled` to clarify this. Maybe rename the `License` class method to `isVersionControlLicensed` and we use this naming convention, WDYT?

Reply:
I agree. I used Enabled because it seems like that was the convention so far within the license class. Will rename it.